times_linux.go

```diff
@@ -26,6 +30,87 @@ type timespec struct {
 	nobtime
 }
 
+type timespecBtime struct {
+	atime
+	mtime
+	ctime
+	btime
+}
+
+// Stat returns the Timespec for the given filename.
+func Stat(name string) (Timespec, error) {
+	var statx unix.Statx_t
+
+	err := unix.Statx(unix.AT_FDCWD, name, unix.AT_EMPTY_PATH|unix.AT_STATX_SYNC_AS_STAT, unix.STATX_ATIME|unix.STATX_MTIME|unix.STATX_CTIME|unix.STATX_BTIME, &statx)
```

Comment:
I usually like to link to the spec somewhere for these more complicated calls so readers know where all this comes from:
https://man7.org/linux/man-pages/man2/statx.2.html